game-service/src/main/java/org/libertybikes/game/round/service/PartyService.java

```diff
@@ -104,6 +104,7 @@ public void deleteParty(String partyId) {
 
     @GET
     @Path("/{partyId}/round")
+    @Produces(MediaType.TEXT_HTML)
```

Comment:
I believe the client frontend depends on this being `APPLICATION_JSON` type, can you instead change the ai-service client to `@Consume` `APPLICATION_JSON`?